packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -382,9 +384,88 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
     action: 'comment',
   });
 
+  const getMetaDescription = (meta: string) => {
+    try {
+      const parsedMeta = JSON.parse(meta);
+      if (getTextFromDelta(parsedMeta)) {
+        return getTextFromDelta(parsedMeta) || meta;
+      } else {
+        return meta;
+      }
+    } catch (error) {
+      return;
+    }
+  };
+
+  const ogTitle =
+    thread?.title?.length > 60
+      ? `${thread?.title?.slice?.(0, 52)}...`
+      : thread?.title;
+  const ogDescription =
+    getMetaDescription(thread?.body || '')?.length > 155
+      ? `${getMetaDescription(thread?.body || '')?.slice?.(0, 152)}...`
+      : getMetaDescription(thread?.body || '');
+  const ogImageUrl = app?.chain?.meta?.iconUrl;
```

Comment:
We should use `app.config` instead to pull from the /status loaded info rather than having to wait for app.chain.

Reply:
actually app.chain is already set by this point, so i think calling it won't be an issue, but lmk if otherwise